spec/controllers/application_controller_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationController do
+  controller do
+    def set_seller_test
+      set_seller
+    end
+
+    def require_seller_login_test
+      require_seller_login
+    end
+  end
+
+  describe "require_seller_login" do
+    before :each do
+      @seller1 = Seller.create(username: "user1", email: "email1@email.com", password: "password1", password_confirmation: "password1")
```

Comment:
It seems like this `@seller1` variable is not needed in the overall before each, but rather should be in the before each in the "seller logged in" section